ach/src/test/java/com/adyen/checkout/ach/internal/ui/model/ACHDirectDebitComponentParamsMapperTest.kt

```diff
@@ -117,6 +118,81 @@ internal class ACHDirectDebitComponentParamsMapperTest {
         assertEquals(expectedAddressParams, params.addressParams)
     }
 
+    @Test
+    fun `when the isStorePaymentFieldVisible  in configuration is false, isStorePaymentFieldVisible in component params should be false`() {
+        val achConfiguration = getAchConfigurationBuilder().apply {
```

Comment:
@onurkaragunlu just noticed this while working on something else
FYI you can use the builder pattern here without the apply:
```kotlin
val achConfiguration = getAchConfigurationBuilder()
    .setShowStorePaymentField(false)
    .build()
```